packages/amplify-migration-e2e/src/assertions.ts

```diff
@@ -156,6 +165,9 @@ async function assertUserPoolResource(projRoot: string, gen1UserPoolId: string,
   const gen2UserPoolId = gen2Meta.auth.user_pool_id;
   const gen2Region = gen2Meta.auth.aws_region;
   const gen2Resource = await getResourceDetails('AWS::Cognito::UserPool', gen2UserPoolId, gen2Region);
+  console.log(gen1Resource, gen2Resource);
+  gen1Resource.UserPoolName = extractUserPoolNamePrefix(gen1Resource.UserPoolName);
```

Comment:
`sandbox` environment will have different user pool names compared to the gen1 environment name